### DISCUSSION TITLE
How to prevent default while lenis is stopped?

### DISCUSSION THREAD

**Discussion by njw275** [darkroomengineering/lenis#281] [Q&A] [answered]
I have an issue where i am stopping lenis with `lenis.stop()` to play an animation and after the animation is finished i use `lenis.start()` but the problem is if you scroll during the animation time then when lenis starts up again there still are some frames where the mousewheel event is still running - is there a way to swallow these and prevent default until you get a new scroll event after lenis has started up again?

**Comment by njw275** [comment]
Or maybe rather with an event like mousewheel when you remove the event listener it stops completely but if you are scrolling and call `lenis.stop()` the scroll event keeps going until you stop scrolling completely - if theres a way to kill the listener for Lenis immediately on stop() call...

**Comment by njw275** [comment, marked as answer]
Found the issue - it was related to CSS overflow

**Reply by dngraphisme** [reply]
Hello @njw275  i have the same problem. can you be more specific please ? Thanks Nicolas

**Reply by njw275** [reply]
My issue was i didnt have the proper CSS implemented correctly as suggested in the README
